tests/ha/test_multi_pod_failure.py

```diff
@@ -2341,3 +2570,158 @@ def test_degraded_mpu_after_kpods_fail(self):
                     "Downloaded the object & verified the checksum.")
 
         LOGGER.info("COMPLETED: Test to verify multipart upload after all K data pods are failed.")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.skip(reason="Not supported in RGW yet")
+    @pytest.mark.tags("TEST-35786")
+    @CTFailOn(error_handler)
+    def test_copy_object_during_kpods_down(self):
+        """
+        Test to Verify copy object during data pods failure till K pods.
+        """
+        LOGGER.info("STARTED: Verify copy object during data pods failure till K pods.")
+
+        bkt_obj_dict = {}
+        output = Queue()
+        bkt_obj_dict[f"ha-bkt-{perf_counter_ns()}"] = f"ha-obj-{perf_counter_ns()}"
+
+        LOGGER.info("Creating s3 account with name %s", self.s3acc_name)
+        resp = self.rest_obj.create_s3_account(acc_name=self.s3acc_name,
+                                               email_id=self.s3acc_email,
+                                               passwd=S3_CFG["CliConfig"]["s3_account"]["password"])
+        assert_utils.assert_true(resp[0], resp[1])
+        access_key = resp[1]["access_key"]
+        secret_key = resp[1]["secret_key"]
+        s3_test_obj = S3TestLib(access_key=access_key, secret_key=secret_key,
+                                endpoint_url=S3_CFG["s3_url"])
+        LOGGER.info("Successfully created s3 account with name %s", self.s3acc_name)
+        self.s3_clean = {'s3_acc': {'accesskey': access_key, 'secretkey': secret_key,
+                                    'user_name': self.s3acc_name}}
+
+        LOGGER.info("Step 1: Create bucket, upload an object to one of the bucket ")
+        resp = self.ha_obj.create_bucket_copy_obj(s3_test_obj=s3_test_obj,
+                                                  bucket_name=self.bucket_name,
+                                                  object_name=self.object_name,
+                                                  bkt_obj_dict=bkt_obj_dict,
+                                                  file_path=self.multipart_obj_path)
+        assert_utils.assert_true(resp[0], resp[1])
+        put_etag = resp[1]
+        LOGGER.info("Step 1: Successfully created bucket, uploaded an object to the bucket")
+
+        bkt_cnt = HA_CFG["copy_obj_data"]["bkt_multi"]
+        for cnt in range(bkt_cnt):
+            rd_time = perf_counter_ns()
+            s3_test_obj.create_bucket(f"ha-bkt{cnt}-{rd_time}")
+            bkt_obj_dict[f"ha-bkt{cnt}-{rd_time}"] = f"ha-obj{cnt}-{rd_time}"
```

Comment:
in step 1, we already copied data to existing value in bkt_obj_dict, I thing we should clear this dict or create a new one. Might get some error.

Reply:
As per discussion with @ApurwaMohite added clear dict at line 2613, also added a comment as to why we need step 1, at line 2603